docs/buildah-bud.md

```diff
@@ -185,6 +185,18 @@ This is a Docker specific option to disable image verification to a Docker
 registry and is not supported by Buildah.  This flag is a NOOP and provided
 soley for scripting compatibility.
 
+**--dns**=[]
+
+Set custom DNS servers
+
+**--dns-option**=[]
+
+Set custom DNS options
+
+**--dns-search**=[]
+
+Set custom DNS search domains
+
```

Comment:
Can you add an example or two?

Reply:
:ok_hand:  will add an example in this patch